### ISSUE TITLE
gwt and smartgwt integration

### ISSUE THREAD

**Issue by GoogleCodeExporter** [googlearchive/gwt-google-apis#425]
```
Found in Release:
gwt-visualization-1.1.0

Detailed description:
My project it's a gwt (2.0.4) project and it's based on smartgwt (2.2) user 
interface container, well the integration between gwt and smartgwt work very 
fine, for example there is a smartgwt container that contains
pie chart as com.google.gwt.visualization.client.visualizations.PieChart (js 
library was loaded using google.load('visualization', '1', {'packages' : 
['piechart']}),
but recently, after upgrading to 
com.google.gwt.visualization.client.visualizations.corechart.PieChart (js 
library was loaded using google.load('visualization', '1', {'packages' : 
['corechart']}), com.google.gwt.visualization.client.visualizations.PieChart 
was deprecated, the UI don't work fine, appears "this.Ob is not an object" 
intead the pie, for verify the behavior I have integrated in my project latest 
visualizationshowcase project and the effect is the same.
In other words
com.google.gwt.visualization.client.visualizations.PieChart integrated in 
smartwgt work fine
but
com.google.gwt.visualization.client.visualizations.corechart.PieChart
don't work,
and any chart in com.google.gwt.visualization.client.visualizations.corechart.* 
don't work.
using any chart in com.google.gwt.visualization.client.visualizations.* any 
chart work very fine.
Tanks in advantage, any aid is appreciate.

Workaround if you have one:


Links to the relevant GWT Developer Forum posts:

```

Original issue reported on code.google.com by `vlui...@tiscali.it` on 15 Oct 2010 at 12:21
